src/lib.rs

```diff
@@ -349,6 +349,8 @@ if_hyper! {
     #[cfg(feature = "__tls")]
     // Re-exports, to be removed in a future release
     pub use tls::{Certificate, Identity};
+    #[cfg(feature = "__rustls")]
+    pub use tls::Crl;
```

Comment:
Since the other re-exports will be removed from the top-level eventually, we probably shouldn't add this one. People should reach for it from `reqwest::tls::...`.

Reply:
You are right. Thank you. I will fix this.